src/index.html

```diff
@@ -7,6 +7,376 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <div class="wrapper">
+      <header class="header wrapper__header">
+        <div class="header__container">
+          <a href="#" class="header__logo">
+            <img src="./images/logo.png" alt="logo">
```

Comment:
![image](https://github.com/mate-academy/layout_catalog/assets/54860641/a5005f7f-6559-4a78-ab1c-3a144e1129ea)

Reply:
Pay attention to comments, the alt "logo" is not meaningful at all.